src/model/database.py

```diff
@@ -27,3 +26,31 @@ def __validate__(schema_data):
     def __serialize_tables__(self):
         for table in self.tables:
             table.serialize()
+
+    def get_table(self, table_name):
+        for table in self.tables:
+            if table.get_name() == table_name:
+                return table
+        raise WrongParameterError("Wrong table entered")
+
+    @staticmethod
+    def get_database_by_name(database_name):
+        return Database(Database.load_database(database_name))
+
+    @staticmethod
+    def load_database(database_name):
```

Comment:
You didn’t follow what I explained in the call.
You don’t need this method.
If there is still misunderstanding, let’s have another call

Reply:
I will stop reviewing till this get fixed